### PR TITLE
added namespace to tankdrivetrain

### PR DESCRIPTION
**Pull request by yutytuty**

### CODE REVIEW

**Review by OfriRosenbaum — CHANGES_REQUESTED** [review]

File: src/main/java/com/spikes2212/command/drivetrains/OdometryDrivetrain.java [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 public abstract class OdometryDrivetrain extends TankDrivetrain {
 
-    public OdometryDrivetrain(MotorController left, MotorController right) {
-        super(left, right);
+    public OdometryDrivetrain(String namespaceName, MotorController left, MotorController right) {
```
[the remark this answers is not in the document]
Comment (yutytuty) [reply src/main/java/com/spikes2212/command/drivetrains/OdometryDrivetrain.java:8]: wat

File: src/main/java/com/spikes2212/command/drivetrains/OdometryDrivetrain.java [hunk 2]
```diff
@@ -23,6 +23,7 @@ public OdometryDrivetrain(MotorController left, MotorController right) {
 
     @Override
     public void periodic() {
+        super.periodic();
```
Comment (OfriRosenbaum) [inline src/main/java/com/spikes2212/command/drivetrains/OdometryDrivetrain.java:26]: first call `getHandler().calculate()` and only then call the `super.periodic()`

File: src/main/java/com/spikes2212/command/drivetrains/TankDrivetrain.java [hunk 3]
```diff
@@ -16,15 +17,18 @@
...
 public class TankDrivetrain extends SubsystemBase {
 
+    protected RootNamespace rootNamespace;
+
     protected MotorController leftController;
     protected MotorController rightController;
     private final DifferentialDrive drive;
 
-    public TankDrivetrain(MotorController left, MotorController right) {
+    public TankDrivetrain(MotorController left, MotorController right, String namespaceName) {
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/command/drivetrains/TankDrivetrain.java:26]: namespace first

**Review by tal-sitton — CHANGES_REQUESTED** [review]

**Review by tal-sitton — APPROVED** [review]

File: src/main/java/com/spikes2212/command/drivetrains/TankDrivetrain.java [hunk 4]
```diff
@@ -16,15 +17,18 @@
  */
 public class TankDrivetrain extends SubsystemBase {
 
+    protected RootNamespace rootNamespace;
+
```
Comment (tal-sitton) [inline src/main/java/com/spikes2212/command/drivetrains/TankDrivetrain.java:21]: why blank line?

**Review by tal-sitton — CHANGES_REQUESTED** [review]

**Review by tal-sitton — APPROVED** [review]

**Pull request merged**